Box2D memory leaks with ropes

The Box2D implementation for HTML  thow the error on web brower
Error: java.lang.RuntimeException: java.lang.AssertionError: End of stack reached, there is probably a leak somewhere

The problem occurs when simulating many ropes, in my case 66, with few does not occur

a similar issue was fixed in jbox2d on 21 Sep 2017

https://github.com/jbox2d/jbox2d/pull/59

LibGDX version 1.9.9

#### Stacktrace

html-0.js:8835 Uncaught Error: java.lang.RuntimeException: java.lang.AssertionError: End of stack reached, there is probably a leak somewhere
    at F7c_g$.X6c_g$ [as createError_0_g$] (html-0.js:79829)
    at F7c_g$.f7c_g$ [as initializeBackingError_0_g$] (html-0.js:79884)
    at F7c_g$.T6c_g$ (html-0.js:79769)
    at F7c_g$.x7c_g$ (html-0.js:79995)
    at new F7c_g$ (html-0.js:80042)
    at Up_g$.Vp_g$ [as execute_0_g$] (html-0.js:8835)
    at html-0.js:184088
    at Okh_g$ (html-0.js:186905)
    at Rkh_g$ (html-0.js:186975)
    at html-0.js:186941
```

#### Please select the affected platforms
- [X ] HTML/GWT

UPDATE: changing the same lines in my LibGDX local repository copy as in https://github.com/jbox2d/jbox2d/pull/59/files solves the problem

--- a/extensions/gdx-box2d/gdx-box2d-gwt/src/com/badlogic/gdx/physics/box2d/gwt/emu/org/jbox2d/dynamics/joints/RopeJoint.java
+++ b/extensions/gdx-box2d/gdx-box2d-gwt/src/com/badlogic/gdx/physics/box2d/gwt/emu/org/jbox2d/dynamics/joints/RopeJoint.java
@@ -101,6 +101,8 @@ public class RopeJoint extends Joint {
       m_u.setZero();
       m_mass = 0.0f;
       m_impulse = 0.0f;
+      pool.pushRot(2);
+      pool.pushVec2(1);
       return;
     }
 

